libnmstate/nm/device.py

```diff
@@ -186,7 +186,9 @@ def _modify_callback(src_object, result, user_data):
 
 
 def _requires_activation(dev, connection_profile):
-    if _mtu_changed(dev, connection_profile):
+    if nmclient.nm_version() < '1.18' and _mtu_changed(
```

Comment:
I propose to use the `packaging` module and do something like this.
```
from packaging import version
...
if version.parse(nmclient.nm_version()) < version.parse('1.18'):
```

What do you think?

Reply:
https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python contains some guidance

Reply:
> https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python contains some guidance

I've been reading about setuptools and maybe in our case it could be useful because NetworkManager does not follow PEP.

Reply:
I do not think that `packaging` is a stdlib and depending on it during runtime may not be a good thing.

Reply:
We have decided to use distutils.version module as it is stdlib.